modules/ref_fb_module/src/CMakeLists.txt

```diff
@@ -23,62 +18,83 @@ set(SRC_Include common.h
                             ${MODULE_HEADERS_DIR}/statistics_fb_impl.h
                             ${MODULE_HEADERS_DIR}/module_dll.h
                             ${MODULE_HEADERS_DIR}/scaling_fb_impl.h
                             ${MODULE_HEADERS_DIR}/dispatch.h
                             ${MODULE_HEADERS_DIR}/trigger_fb_impl.h
-                            ${MODULE_BINARY_HEADER_DIR}/arial.ttf.h
+                            ${MODULE_HEADERS_DIR}/classifier_fb_impl.h
                             module_dll.cpp
-                            ref_fb_module_impl.cpp
-                            renderer_fb_impl.cpp
                             power_fb_impl.cpp
                             statistics_fb_impl.cpp
                             scaling_fb_impl.cpp
                             classifier_fb_impl.cpp
-                            trigger_fb_impl.cpp
-)
+                            trigger_fb_impl.cpp)
 
-source_group("polyline" FILES ${MODULE_HEADERS_DIR}/polyline.h
-                              ${MODULE_HEADERS_DIR}/polyline_impl.h
-                              ${MODULE_HEADERS_DIR}/polyline_impl_generic.h
-                              ${MODULE_HEADERS_DIR}/polyline_impl_line.h
-                              polyline.cpp
-                              polyline_impl.cpp
-                              polyline_impl_generic.cpp
-                              polyline_impl_line.cpp
+if (DAQMODULES_REF_FB_MODULE_ENABLE_RENDERER)
+    set(MODULE_FILES ${MODULE_FILES} ${MODULE_HEADERS_DIR}/renderer_fb_impl.h
+                                     ${MODULE_BINARY_HEADER_DIR}/arial.ttf.h
+                                     renderer_fb_impl.cpp
+    )
+
+    source_group("polyline" FILES ${MODULE_HEADERS_DIR}/polyline.h
+                                  ${MODULE_HEADERS_DIR}/polyline_impl.h
+                                  ${MODULE_HEADERS_DIR}/polyline_impl_generic.h
+                                  ${MODULE_HEADERS_DIR}/polyline_impl_line.h
+                                  polyline.cpp
+                                  polyline_impl.cpp
+                                  polyline_impl_generic.cpp
+                                  polyline_impl_line.cpp
 )
+endif()
+
+source_group("module" FILES ${MODULE_FILES})
 
 add_library(${LIB_NAME} SHARED ${SRC_Include}
                                ${SRC_Srcs}
-                               ${MODULE_BINARY_HEADER_DIR}/arial.ttf.h
+                               
 )
 
 add_library(${SDK_TARGET_NAMESPACE}::${LIB_NAME} ALIAS ${LIB_NAME})
 
-target_link_libraries(${LIB_NAME} PUBLIC daq::opendaq
-                                  PRIVATE sfml-graphics
-                                          sfml-window
-                                          date::date
-)
+target_link_libraries(${LIB_NAME} PUBLIC daq::opendaq)
+
+if (DAQMODULES_REF_FB_MODULE_ENABLE_RENDERER)
+    target_compile_definitions(${LIB_NAME} PRIVATE ENABLE_RENDERER)
```

Comment:
I'd prefix definitions with `OPENDAQ_` so we're sure that it doesn't clash with anything